fee-monitor/src/index.ts

```diff
@@ -183,7 +183,14 @@ async function main() {
     for (;;) {
         console.log();
         console.log(`BlockNumber: ${blockNumber}`);
+        const cloneChecker = (instance: DynamicChecker): DynamicChecker => {
+            const copy = new (instance.constructor as { new (): DynamicChecker })();
```

Comment:
How this code is different from `const copy = new DynamicChecker()`?

Reply:
Ah, they are same. That's because I modified the code from a more general one. I'll fix it.